src/layouts/index.js

```diff
@@ -13,7 +13,6 @@ const menuItems = [
   { link: '/', text: 'Home', exact: true },
   { link: 'https://ti.to/web-audio-conference/wac-2018', external: true, text: 'Tickets', extraClassName: 'header__emph' },
   { link: '/schedule', text: 'Schedule' },
-  { link: '/program', text: 'Program' },
```

Comment:
Could you add a redirect from `/program` to `/schedule` ?

Reply:
Good idea @janmonschke, do you know how this works in Gatsby? I could duplicate the schedule page at the /program route but that doesn't feel right.

Reply:
I found out how to do it. It's pretty simple if you know what to google for. :-)